password-strength-checker/enabled.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  name: 'the password strength checker is enable',
+  hypothesis: 'the password strength checker will prevent users from signing up with insecure passwords.',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uniqueUserId', 'isMetricsEnabled', 'forcePasswordCheck'],
```

Comment:
Maybe rename `forcePasswordCheck` to `forcePasswordStrengthCheck` for consistency.